ext/reflection/php_reflection.h

```diff
@@ -50,6 +50,16 @@ extern PHPAPI zend_class_entry *reflection_fiber_ptr;
 
 PHPAPI void zend_reflection_class_factory(zend_class_entry *ce, zval *object);
 
+/* Struct for parameters */
+typedef struct _parameter_reference {
+	uint32_t offset;
+	bool required;
+	struct _zend_arg_info *arg_info;
+	zend_function *fptr;
+} parameter_reference;
```

Comment:
Could you please prefix the names with `reflection_` then as it's in a public header now?

Reply:
Sure, that seems like a good idea. Thanks for the suggestion. Should the prefix actually be `zend_reflection_`?

Reply:
No, zend is for Zend/ (generally).